docs/source/python/api/arrays.rst

```diff
@@ -86,6 +86,10 @@ may expose data type-specific methods or properties.
    ExtensionArray
    FixedShapeTensorArray
    OpaqueArray
+   JsonArray
+   UuidArray
+   Bool8Array
+   Decimal256Array
```

Comment:
JSON, UUID, and Bool8 are all canonical extension types, so those array classes belong here at the end of the list. But Decimal256 is a top-level Arrow type, so `Decimal256Array` belongs higher up in this list, immediately below `Decimal128Array`. Could you move it there please? Thanks